build/pause/Makefile

```diff
@@ -18,7 +18,7 @@ REGISTRY ?= staging-k8s.gcr.io
 IMAGE = $(REGISTRY)/pause
 IMAGE_WITH_ARCH = $(IMAGE)-$(ARCH)
 
-TAG = 3.2
+TAG = 4.0
```

Comment:
what changed in pause? bumping pause is a headache

Reply:
From [Slack](https://kubernetes.slack.com/archives/C2C40FMNF/p1588292996133600?thread_ts=1588288957.129100&cid=C2C40FMNF), we decided to bump the major versions to represent them being moved over to K8s Infra.

Content-wise, I'm not sure there are discernible changes.

What's involved in the pause bump?
If it's a really big hassle, I'm fine with backing that out.
Here, I'm more concerned with getting the `debian-base` and `debian-iptables` images in a buildable state on K8s Infra.

Reply:
bumping pause requires all the deployment tools (kubeadm, ...) + dockershim (kubelet) + container runtimes (containerd, crio) to need to update to the new image and ideally align on this.

Reply:
if there are content changes it needs bumping though ...
given that it's just switching build systems, I would expect `3.3`, semantically we're not altering compatibility are we?

Reply:
@BenTheElder -- Good point. We're not, AFAICT.
I've set the `pause` image version to `3.3`.

Also, brought the `debian-base` version to `v2.1.0`.